test-tools/proof-explorer/src/Root.tsx

```diff
@@ -211,12 +211,13 @@ async function submitProof(
         return;
     }
     console.log(proof.toString());
+    console.log(proof);
     const resp = await fetch(`${getVerifierURL()}/v0/verify`, {
         method: 'POST',
         headers: {
             'Content-Type': 'application/json',
         },
-        body: proof.toString(),
+        body: JSON.stringify(proof),
```

Comment:
Will this work properly also when there are bigints in the proof?

Reply:
Not if they can be in the proof object, no. I will take a look at it.

Reply:
This was part of the "Hacky" fix commit message, I just forgot to revisit it...

Reply:
There was also an issue with the serialization of parameters for WalletConnect which I have also fixed now. I will update the dapp-libraries accordingly (and the CryptoX wallet to match).